commit 2aa7b47354bdf5863071c8b479d29c99aad05ecb
Author: Johannes Meixner <jsmeix@suse.com>
Date:   Fri Jul 24 13:02:45 2020 +0200

    Update 240_reassign_luks_keyfiles.sh
    
    Use ReaR specific TMP_DIR (not TMPDIR or hardcoded /tmp)

diff --git a/usr/share/rear/finalize/GNU/Linux/240_reassign_luks_keyfiles.sh b/usr/share/rear/finalize/GNU/Linux/240_reassign_luks_keyfiles.sh
index d989c3fb..358f3950 100644
--- a/usr/share/rear/finalize/GNU/Linux/240_reassign_luks_keyfiles.sh
+++ b/usr/share/rear/finalize/GNU/Linux/240_reassign_luks_keyfiles.sh
@@ -24,9 +24,9 @@ awk '
 while read target_name source_device original_keyfile; do
     Log "Re-assigning keyfile $original_keyfile to LUKS device $target_name ($source_device)"
 
-    # The scheme for generating a temporary keyfile path must be the same here and in the 'layout/prepare' stage.
-    temp_keyfile="${TMPDIR:-/tmp}/LUKS-keyfile-$target_name"
-    [ -f "$temp_keyfile" ] || BugError "temporary keyfile $temp_keyfile not found"
+    # The scheme for generating a temporary keyfile path must be the same here and in the 'layout/prepare' stage:
+    temp_keyfile="$TMP_DIR/LUKS-keyfile-$target_name"
+    test -f "$temp_keyfile" || BugError "temporary LUKS keyfile $temp_keyfile not found"
 
     target_keyfile="$TARGET_FS_ROOT/$original_keyfile"
 
